### ISSUE TITLE
Documentation in manual for --stream is incorrect

### ISSUE THREAD

**Issue by gnuvince** [jqlang/jq#1279]
The manual states:

> Parse the input in streaming fashion, outputing arrays of path and leaf values (scalars and empty arrays or empty objects). For example, "a" becomes [[],"a"], and [[],"a",["b"]] becomes [[0],[]], [[1],"a"], and [[1,0],"b"]

However, this is incorrect, the path for `"b"` is `[2,0]`.

```
~$ echo '[[],"a", ["b"]]'|jq -Mc --stream .
[[0],[]]
[[1],"a"]
[[2,0],"b"]
[[2,0]]
[[2]]
```

**Comment by nicowilliams** [comment]
Thanks for this report!

**Comment by itchyny** [comment]
Resolved by #2387, so closing.
